src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -3914,6 +3916,43 @@ def setUp(self):
         self._use_http_authorization(username=django_settings.API_EXT_USER['username'],
             password=django_settings.API_EXT_USER['password'])
 
+    def test_external_service_can_not_read_all_metadata_in_other_catalog(self):
+        ''' External service should get the same output from someone elses catalog than anonymous user '''
+        # create a catalog that does not belong to our external service
+        dc2 = DataCatalog.objects.get(pk=2)
+        dc2.catalog_json['identifier'] = 'Some other catalog'
+        dc2.catalog_record_services_create = 'metax'
+        dc2.catalog_record_services_edit = 'metax'
+        dc2.catalog_record_services_read = 'metax'
+        dc2.catalog_record_group_read = 'metax'
```

Comment:
I think the test would be even better, if from these permissions only the ``catalog_record_services_read`` is defined, since this is the field that is being tested here. If for some reason there became a bug that the permissions are checked based on some other field, then this testcase would still pass.

Reply:
Thanks for positive feedback :) fixed the test